docs/user/discover.asciidoc

```diff
@@ -275,8 +270,17 @@ image:images/discover-maps.png[Map containing documents]
 [[share-your-findings]]
 === Share your findings
 
-To share your findings with a larger audience, click *Share* in the *Discover* toolbar. For detailed information about the sharing options, refer to <<reporting-getting-started,Reporting>>.
+To share your findings with a larger audience, click *Share* in the *Discover* toolbar.
+For detailed information about the sharing options, refer to <<reporting-getting-started,Reporting>>.
+
+[float]
+[[alert-from-Discover]]
+=== Generate alerts
 
+To create a rule from *Discover*,
+click *Alerts* in the toolbar. Then build the data view, query,
+and filters that match the data for which you want the alert notification.
+For detailed information, refer to <<alerting-getting-started, Alerting>>.
```

Comment:
We should advise here to first use Discover to choose the data view, query, filters to get the data one would love to have alerts when new data is inserted into Elasticsearch. Because the latest query sent to Elasticsearch is used to prefill the form in the alert flyout. Happy to demo this if something's unclear about that.